File: OOAD/tictactoe-app/src/components/Player.java
package components;

import components.enums.Mark;

public class Player {
	private Mark mark;
	private String name;

	public Player(String name,Mark mark) {
		this.setMark(mark);
		this.name=name;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public Mark getMark() {
		return mark;
	}

	public void setMark(Mark mark) {
		this.mark = mark;
	}
	
	@Override
	public String toString(){
		return this.name;
	}
}

File: C#/MVC/TicTacToe_Layered_With_Unit_Testing-App/TicTacToe_Lib/Components/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TicTacToe_Lib.Enum;

namespace TicTacToe_Lib.Components
{
     public class Player
    {
        private Mark mark;
        private string name;

        public Player(String name, Mark mark)
        {
            this.setMark(mark);
            this.name = name;
        }

        //public string Name { get { return name; } set{name= value;} }
        public string getName()
        {
            return name;
        }

        public void setName(String name)
        {
            this.name = name;
        }

        public Mark getMark()
        {
            return mark;
        }

        public void setMark(Mark mark)
        {
            this.mark = mark;
        }

        public override string ToString()
        {
            return this.name;
        }
    }
}
